test/e2e/network_partition.go

```diff
@@ -391,7 +391,7 @@ var _ = framework.KubeDescribe("Network Partition [Disruptive] [Slow]", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			pst := statefulSetTester{c: c}
-			pst.saturate(ps)
```

Comment:
`saturate` does the step-wise initialization of pets when we have the debug annotation set. Since we don't have it set here, we can just wait for all the replicas to start running? I'm not sure why we should call saturate here.